frontend/src/component/Activity/FileActivity.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import ContentActivityHeader from './ContentActivityHeader.jsx'
+import ContentActivityFooter from './ContentActivityFooter.jsx'
+
+export const FileActivity = props => {
```

Comment:
It is just for files or all contents?
```suggestion
export const ContentWithPreviewActivity = props => {
```